### PR TITLE
Skatepark: query loop block pattern

### PR DESCRIPTION
**Pull request by MaggieCabrera**
#### Changes proposed in this Pull Request:

As discussed on slack, this PR brings in the index.html query loop as a pattern since we didn't have any pattern for the block:

Screenshot 2021-08-19 at 17 39 33

### CODE REVIEW

File: skatepark/inc/patterns/blog-posts.php [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+	'title'      => __( 'Blog posts', 'skatepark' ),
+	'categories' => array( 'skatepark' ),
+	'content'    => '<!-- wp:query {"query":{"perPage":6,"pages":0,"offset":0,"postType":"post","categoryIds":[],"tagIds":[],"order":"desc","orderBy":"date","author":"","search":"","exclude":[],"sticky":"exclude","inherit":false},"displayLayout":{"type":"flex","columns":3},"align":"wide"} -->
+	<div class="wp-block-query alignwide"><!-- wp:post-template -->
+	<!-- wp:group {"style":{"spacing":{"padding":{"top":"30px","right":"30px","bottom":"30px","left":"30px"}}},"layout":{"inherit":false}} -->
+	<div class="wp-block-group" style="padding-top:30px;padding-right:30px;padding-bottom:30px;padding-left:30px"><!-- wp:separator {"className":"is-style-wide"} -->
+	<hr class="wp-block-separator is-style-wide"/>
+	<!-- /wp:separator -->
+	
+	<!-- wp:post-featured-image {"style":{"color":{"duotone":["#000","#B9FB9C"]}}} /-->
```
Comment (kjellr) [inline skatepark/inc/patterns/blog-posts.php]: We should make sure `isLink` is true for the featured images here (same for the actual template itself)

**Comment by melchoyce** [comment]
What's up with the link placeholder alignment here? 👀 

image

**Comment by MaggieCabrera** [comment]
Added image link to the pattern and the template and fixed the read more text (the text-indent from the except was being applied to the paragraph that contains the read more link)

**Review by mikachan — COMMENTED** [review]
This is looking good on my side! The only things I can spot compared to the design is an underline on the dates, or maybe these should be links?

And should the 'read more' link have a heavier font-weight, maybe `500`?

Here's the design I'm looking at:

Screenshot 2021-08-20 at 12 24 14

**Comment by kjellr** [comment]
Yeah, I'll second the notes from @mikachan. Also, the vertical spacing between all the post meta (featured image, title, excerpt, etc.) is all much greater than it is in the comps. Can we make that a little closer to the mockups? 

Comps|This PR
---|---
Screen Shot 2021-08-20 at 8 54 40 AM|Screen Shot 2021-08-20 at 8 54 54 AM

**Comment by jffng** [comment]
> Also, the vertical spacing between all the post meta (featured image, title, excerpt, etc.) is all much greater than it is in the comps. Can we make that a little closer to the mockups?

I've noticed this too during most of my attempts to make a pattern in Skatepark, and I think it has to do with this: https://github.com/Automattic/themes/blob/trunk/skatepark/child-theme.json#L203

Should we reduce the value of this variable to something closer to the vertical spacing between the text in the comps?

**Comment by MaggieCabrera** [comment]
I reduced the vertical spacing from 45 to 40px, I think it's more fitting to the design. The biggest problem about the spacing is that we were applying the same distance between the headings and the rest of the content and between every paragraph and the rest. According to figma, the distance between paragraphs should be 20px instead so I override that for Skatepark on the CSS and that makes the theme look like the design. 

I've created a [separate issue to support RTL](https://github.com/Automattic/themes/issues/4453) with the arrow so that we don't block this PR with that issue.

File: skatepark/inc/patterns/blog-posts.php [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+	<!-- wp:group {"style":{"spacing":{"padding":{"top":"30px","right":"30px","left":"30px"}}},"layout":{"inherit":false}} -->
+	<div class="wp-block-group" style="padding-top:30px;padding-right:30px;padding-left:30px"><!-- wp:separator {"className":"is-style-wide"} -->
+	<hr class="wp-block-separator is-style-wide"/>
+	<!-- /wp:separator -->
+	
+	<!-- wp:post-featured-image {"isLink":true,"style":{"color":{"duotone":["#000","#B9FB9C"]}}} /-->
+	
+	<!-- wp:post-title {"isLink":true,"fontSize":"normal"} /-->
+	
+	<!-- wp:post-excerpt {"moreText":"Continue reading","className":"is-style-indented-post-excerpt"} /-->
```
Comment (scruffian) [inline skatepark/inc/patterns/blog-posts.php]: This text should be translated.

Comment (MaggieCabrera) [reply skatepark/inc/patterns/blog-posts.php]: Great catch thank you!

File: skatepark/block-templates/index.html [hunk 3]
```diff
@@ -5,12 +5,12 @@
...
-	<div class="wp-block-group" style="padding-top:30px;padding-right:30px;padding-bottom:30px;padding-left:30px"><!-- wp:separator {"className":"is-style-wide"} -->
+	<!-- wp:group {"style":{"spacing":{"padding":{"top":"30px","right":"30px","left":"30px"}}},"layout":{"inherit":false}} -->
+	<div class="wp-block-group" style="padding-top:30px;padding-right:30px;padding-left:30px"><!-- wp:separator {"className":"is-style-wide"} -->
 	<hr class="wp-block-separator is-style-wide"/>
 	<!-- /wp:separator -->
 	
-	<!-- wp:post-featured-image {"style":{"color":{"duotone":["#000","#B9FB9C"]}}} /-->
+	<!-- wp:post-featured-image {"isLink":true,"style":{"color":{"duotone":["#000","#B9FB9C"]}}} /-->
 	
 	<!-- wp:post-title {"isLink":true,"fontSize":"normal"} /-->
```
Comment (scruffian) [inline skatepark/block-templates/index.html:16]: Should we add the same more text to this template?

Comment (MaggieCabrera) [reply skatepark/block-templates/index.html:16]: it's not translatable there...

Comment (scruffian) [reply skatepark/block-templates/index.html:16]: I know, but without it there's no link at all. I guess that's still better than a link that's not translated.

**Review by scruffian — APPROVED** [review]
LGTM

**Pull request merged**